src/django/api/mail.py

```diff
@@ -336,6 +348,31 @@ def send_report_result(report):
         subj_template.render().rstrip(),
         text_template.render(report_dictionary),
         settings.DEFAULT_FROM_EMAIL,
-        [settings.NOTIFICATION_EMAIL_TO],
+        [report.reported_by_contributor.admin.email],
```

Comment:
This is an existing bug - the email should have been being sent to the contributor admin